### DISCUSSION TITLE
What are the data egress charges ?

### DISCUSSION THREAD

**Discussion by sksar** [planetscale/discussion#374] [FAQs] [answered]
I understand that planet-scale is hosted at AWS, which has data egress charges.

So, I was wondering, what are the data egress charges for a planet scale database?
Are there any? Or it is not charged as of now?

I could not find any suitable answer, and I am looking for some clarification.

**Comment by fooforge** [comment, marked as answer]
Hey @sksar,

Good question!

We do not charge separately for AWS' data egress/transfer fees. Technically, these are a cost factor for us as databases on the Scaler plan (or higher) come with instances in at least two different availability zones, but it's already factored in.

Our Scaler plan starts at $29 per month per database, and for usage and storage above the included row reads, row writes, and storage we charge:

* $2.50/mo per GB
* $1/mo per 1 billion rows read
* $1.50/mo per 1 million rows written

Storage for the automatic backups we run for every branch is free, while storage for additional manual or user-scheduled backups is currently billed at $0.023 per GB per month.

For a complete overview (or if somebody stumbles upon this thread in the future), please also consult [our pricing page](https://planetscale.com/pricing).

**Reply by sksar** [reply]
Thanks that clarifies my doubts.
